frontend/src/containers/Header/index.tsx

```diff
@@ -1,12 +1,19 @@
+import { Button } from '@mui/material';
+import { useAppSelector } from 'hooks/redux';
 import { FC } from 'react';
-import { Button } from 'components';
+import { useNavigate } from 'react-router';
+import { NavLink } from 'react-router-dom';
 
 import s from './Header.module.scss';
 
 export const Header: FC = () => {
+  const { user } = useAppSelector(state => state.userReducer);
+  const navigate = useNavigate();
+
   return (
     <div className={s.headerWrapper}>
-      <Button onClick={() => console.log('click')}>button</Button>
+      <h1><NavLink to='/'> DND Mentor</NavLink></h1>
```

Comment:
If you want to use h1 you may use Mui Typography component with props value variant='h1' or custom component TextField and props for h1 too